contents/handbook/growth/sales/product-led-sales.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Product-led Sales
+sidebar: Handbook
+showTitle: true
+---
+
+A large proportion of our paying customer base sign up to a paid plan without ever talking to the Sales team.  We don't want to force these customers through a sales process if they don't need it, but we also know that having a human to help them through the process on hand is likely to maximize the chances of retaining them as a paying customer long term.
+
+> We think that product-led sales is going to be extremely important to how PostHog goes, so we will be iterating on this process a lot as we learn. Feel free to edit this page at any time with things that you have found work/don't work. 
+
+## Product-led lead generation
+
+Product-led leads can be generated in different ways:
+
+1. Customers who have an active subscription in Stripe, and are forecasted to pay a first time bill of $500 or more.
+2. Customers who have a high [lead score](/handbook/growth/sales/lead-scoring) (>=20), and have more than 1 user in the organization, but haven't yet subscribed.
+
+They follow the [normal territory assignment rules](https://posthog.com/handbook/growth/sales/crm#how-we-do-lead-assignments) in Salesforce. 
+
+## Working with the customer
+
+Just as with the [inbound sales process](/handbook/growth/sales/new-sales), it's on you to decide how you qualify the lead.  If you think they have potential to end up paying more than $20k a year then you should reach out to introduce yourself and offer help.  As they have likely done a lot of research themselves, they may not need a demo so a 30-minute [discovery](/handbook/growth/sales/new-sales#maximizing-your-chance-of-success) is probably more appropriate here.
+
+If it's a viable opportunity then you should convert the lead to an [opportunity](/handbook/growth/sales/crm#opportunities) and then follow the [New sales process](/handbook/growth/sales/new-sales). Bear in mind that you can join it at any point depending on where the customer is at in their buying journey (e.g. you might skip product evaluation if they are ready to buy).
+
+Even if after speaking with them you think they may end up at $20k+, you should educate them on how to get help, as well as the value of adding our Teams and Enterprise plans.
```

Comment:
Side note: we should have somewhere public we can link to which says what Teams and Enterprise actually contain from a Sales perspective (setup calls, training etc.)